custom_model_runner/setup.py

```diff
@@ -70,5 +71,5 @@
     },
     install_requires=requirements,
     extras_require=extras_require,
-    python_requires=">=3.8,<3.12",
+    python_requires=">=3.8,<3.13",
```

Comment:
Do we know that we don't support newer versions of Python? Not to increase the scope of this work too much already but did you try running up to the latest and adding that to the matrix?

Reply:
Do you mean 3.13? I didn't tried. I can try, but on another project there was some weird problems with Python 3.13 image. I can try in the next PR

Reply:
Trying here https://app.harness.io/ng/account/oP3BKzKwSDe_4hCFYw_UWA/module/ci/orgs/Custom_Models/projects/datarobotusermodels/pipelines/base_checks/executions/j2oTiFeSQMCCIbQ7DLqxtQ/pipeline?stage=edvsTRj4SQewtOMxlfWHcA